edgetest/core.py

```diff
@@ -25,6 +25,8 @@ class TestPackage:
         The name of the conda environment to create/use.
     upgrade : list
         The list of packages to upgrade
+    lower : list
+        The list of packages to install lower bounds.
```

Comment:
Documentation note: reading the utilities, this is a list of the packages to be lowered alongside the lower bound, correct? I.e.

```python
[
    "pandas==1.5.1",
]
```

instead of

```python
["pandas",]
```

Should we add that to the docs where `lower` is an input? It's different than the format of ``upgrade``.

Reply:
Yeah that's a good catch. In the config, it's just like `upgrade`, but in `TestPackage`, the config is already parsed to provide the package to be lowered alongside the lower bound.

Reply:
Yeah not a huge deal since it's not user facing. Might save us some confusion down the road though.